Vagrantfile

```diff
@@ -9,10 +9,9 @@ Vagrant.configure("2") do |config|
 
 	# The following will not work on case-insensitive filesystems like those provided by
 	# Windows and macOS, as the Linux kernel source tree has files that differ only by case.
-	# If your host is Linux, you can uncomment this line to share the kernel source
-	# directory with your host for easy editing.
-
-	# config.vm.synced_folder "./linux-source", "/home/vagrant/linux-source"
+        unless Vagrant::Util::Platform.windows?
```

Comment:
Better to use fs_case_sensitive, I think

http://www.rubydoc.info/github/mitchellh/vagrant/Vagrant/Util/Platform#fs_case_sensitive%3F-class_method

Reply:
I don't like the hacky way that method works, but surewhynot

Reply:
fixed